docs/admin-guide/index.md

```diff
@@ -11,7 +11,7 @@ myst:
 
 # Admin guide
 
-In this part of the documentation, you can find how to install, operate, configure, and deploy Plone.
+In this part of the documentation, you can find how to install, operate and configure Plone.
```

Comment:
@MrTango Deployment belongs in the admin guide, we just haven't gotten to moving it yet. I'm okay with this edit in the meantime.

Reply:
ok, didn't know that. If we have it in the admin guide, then probably still not mixed with installation right.
As with all the options and details, it get;s a bit to much.

Reply:
Just to add to this conversation, there is a longer term goal to implement Guides as shown in https://pydata-sphinx-theme.readthedocs.io/en/stable/ and https://numpy.org/doc/stable/release.html in their top horizontal navigation. We're in the first phase of reorganization, and that may be part of a second or third phase.